Automattic/woocommerce-payments
# feat: UPE preview onboarding first step

Helps with https://github.com/Automattic/woocommerce-payments/issues/2170

#### Changes proposed in this Pull Request









I decided to split a bigger PR. Other steps of the onboarding flow will be done separately.
This just makes some UI changes on the first onboarding step.
No functionality has been changed, just text and CSS has been updated - no tests have been added.

New visual:
![Screen Shot 2021-07-14 at 10 54 34 AM](https://user-images.githubusercontent.com/273592/125653170-b6b923e0-770c-4c86-a3fc-184bfd635d79.png)


#### Testing instructions





- Ensure you have `_wcpay_feature_account_overview_task_list` with `'1'` in your `wp_options`
- Ensure "Enable UPE settings changes" is enabled
- Ensure "Enable UPE checkout" is disabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- UI is updated with 2 boxes

-------------------
